Documentation/ContentObjects/Hmenu/Index.rst

```diff
@@ -1650,7 +1650,7 @@ generated from this array, which was returned from the function
      20:                      ),
      21:                      array(
      22:                          'title' => 'New York Office',
-     23:                          '_OVERRIDE_HREF' => 'http://www.example.com',
+     23:                          '_OVERRIDE_HREF' => 'https://www.example.com',
```

Comment:
No `.org` here?

Reply:
The menu entry is explicitly overwriting the default domain, therefore i chose to use the next domain in the list.

Reply:
OK, thanks.